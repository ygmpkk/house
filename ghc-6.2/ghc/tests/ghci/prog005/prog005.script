:unset +s
:unset +t
:set -package lang -fglasgow-exts
:l Parser
parser (lexer "a>b")
