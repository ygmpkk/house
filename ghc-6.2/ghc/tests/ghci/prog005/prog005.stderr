Compiling Parser           ( Parser.hs, interpreted )
state: 0,	token: 1,	action: shift, enter state 2
state: 2,	token: 2,	action: reduce (rule 1), goto state 3
state: 3,	token: 2,	action: shift, enter state 4
state: 4,	token: 1,	action: shift, enter state 2
state: 2,	token: 4,	action: reduce (rule 1), goto state 7
state: 7,	token: 4,	action: reduce (rule 2), goto state 3
state: 3,	token: 4,	action: accept.
